src/common/wifi/core/include/microsoft/net/wifi/IAccessPointController.hxx

```diff
@@ -60,6 +61,17 @@ struct IAccessPointController
      */
     virtual Ieee80211AccessPointCapabilities
     GetCapabilities() = 0;
+
+    /**
+     * @brief Set the PHY type of the access point.
+     *
+     * @param phyType The PHY type to be set
+     * @return true
+     * @return false
+     *
+     */
+    virtual bool
+    SetPhyType(Dot11PhyType phyType) = 0;
```

Comment:
See comments in `AccessPointControllerLinux.[hxx|cxx]` regarding layering dependencies and use of `Ieee80211*` types instead of the protobuf types in this layer.